analyticConfigurations/computed_metric-elastic.search.json

```
@@ -300,6 +300,16 @@
           ],
           "fqn": "netuitive.linux.elasticsearch.indices.${1}.warmer.avg_per_sec"
         }
+      },
+      {
+        "match": "netuitive.linux.elasticsearch.jvm.gccollectionpersecond",
```

Could we make "collection" plural?
```suggestion
        "match": "netuitive.linux.elasticsearch.jvm.gccollectionspersecond",
```